OpenEdXMobile/src/main/java/org/edx/mobile/util/DataBindingHelperUtils.kt

```diff
@@ -75,17 +81,21 @@ class DataBindingHelperUtils {
                     }
 
                     isViewVisible(childView.title, item.title.isNullOrBlank().not())
```

Comment:
Can we replace it with 
`if (item.title.isNotBlank())
                        childView.title.visibility = View.VISIBLE`